AdobeStockImage/etc/di.xml

```diff
@@ -47,4 +47,11 @@
             </argument>
         </arguments>
     </type>
+    <type name="Magento\MediaGalleryUi\Model\ImagesIndexer">
```

Comment:
An interface should be created in MediaGalleryUiApi module and used in di instead of actual implementation class